README.md

```diff
@@ -442,6 +442,7 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 - :wrench: [Open Source Reverse Engineering Platform](https://github.com/radareorg/cutter)
 - :video_camera: [Penetration Testing Bootcamp](https://hackersploit.org/penetration-testing-bootcamp/)
 - :bulb: [Practical-Ethical-Hacking-Resources](https://github.com/TCM-Course-Resources/Practical-Ethical-Hacking-Resources)
+- :video_camera: [Complete ethical hacking course](https://www.youtube.com/watch?v=tHd8k54kVs8&list=PLBf0hzazHTGOEuhPQSnq-Ej8jRyXxfYvl)
```

Comment:
Replace with the URL to the playlist

Reply:
I have made the changes , please check .